snet_cli/utils.py

```diff
@@ -123,6 +124,18 @@ def walk_imports(entry_path):
     return seen_paths
 
 
+def get_contract_json(interface_path, implementation_path=None):
```

Comment:
Here are the first several invocations of this function in this code review:

```
get_contract_json(Path(__file__).absolute().parent.joinpath("resources", "contracts", "Agent.json"))
get_contract_json(Path(__file__).absolute().parent.joinpath("resources", "contracts", "AgentFactory.json"))
contract_json = get_contract_json(Path(__file__).absolute().parent.joinpath("resources", "contracts", {}.json".format(contract_name)))
contract_json = get_contract_json(
        Path(__file__).absolute().parent.joinpath("resources", "contracts", "IRegistry.json"),
        Path(__file__).absolute().parent.joinpath("resources", "contracts", "RegistryImpl.json")
    )
token_contract_json = get_contract_json(Path(__file__).absolute().parent.joinpath("resources", "contracts", "SingularityNetToken.json"))
job_contract_json = get_contract_json(Path(__file__).absolute().parent.joinpath("resources", "contracts", "Job.json"))
```

or, with keywords specified:

```
get_contract_json(interface_path=Path(__file__).absolute().parent.joinpath("resources", "contracts", "Agent.json"), implementation_path=None)
get_contract_json(interface_path=Path(__file__).absolute().parent.joinpath("resources", "contracts", "AgentFactory.json"), implementation_path=None)
contract_json = get_contract_json(interface_path=Path(__file__).absolute().parent.joinpath("resources", "contracts", {}.json".format(contract_name)), implementation_path=None)
contract_json = get_contract_json(
        interface_path=Path(__file__).absolute().parent.joinpath("resources", "contracts", "IRegistry.json"),
        implementation_pathPath(__file__).absolute().parent.joinpath("resources", "contracts", "RegistryImpl.json")
    )
token_contract_json = get_contract_json(interface_path=Path(__file__).absolute().parent.joinpath("resources", "contracts", "SingularityNetToken.json"), implementation_path=None)
job_contract_json = get_contract_json(interface_path=Path(__file__).absolute().parent.joinpath("resources", "contracts", "Job.json"), implementation_path=None)
```

Notice that in all but one case, the value for `interface_path` is actually the full implementation (e.g. AgentFactory.json, Job.json, Agent.json, etc.). The semantics of this function are off.

Reply:
Killed: this function was unnecessary